src/routes/apiRouter.js

```diff
@@ -1,10 +1,11 @@
 import express from 'express'
 import Client from '@helium/http'
 import { errorResponse, successResponse } from '../helpers'
-import { getCache, getHexCache, setHexCache } from '../helpers/cache'
+import { getCache, setCache, getHexCache, setHexCache } from '../helpers/cache'
 import { redisClient, timestampRange, aggregation } from '../helpers/redis'
 import { fetchCitySearchGeometry } from '../helpers/cities'
 import { getGeo } from '../helpers/validators'
+const cors = require('cors')
```

Comment:
Can you use ES6 import style?

Reply:
Not sure. The rest of the app isn't, so I just went with `require`.

Reply:
See `src/app.js` for example